packages/graphile-build/src/makeNewBuild.d.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  PgProc,
+  PgType,
+  PgClass,
+  PgAttribute,
+  PgConstraint,
+} from "graphile-build-pg";
+import { GraphQLNamedType } from "graphql";
+
+export interface Build {
+  options: any;
+  graphileBuildVersion: any;
+  versions: any;
+  hasVersion(
+    packageName: string,
+    range: string,
+    options?: { includePrerelease?: boolean }
+  ): boolean;
+  graphql: typeof import("graphql");
+  parseResolveInfo: any;
+  simplifyParsedResolveInfoFragmentWithType: any;
+  getSafeAliasFromAlias: any;
+  /** @deprecated */
+  getAliasFromResolveInfo: any;
+  getSafeAliasFromResolveInfo: any;
+  resolveAlias(data: any, _args: any, _context: any, resolveInfo: any): any;
+  addType(type: GraphQLNamedType, origin?: string): void;
```

Comment:
This should be the class type: GraphQLConstructorType or whatever.

```suggestion
  addType(type: GraphQLConstructorType, origin?: string): void;
```